README.md

```diff
@@ -735,23 +759,24 @@ To add a new block type implementation, make sure to examine and amend:
-[`EncodeCSV`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.EncodeCsv.html
-[`EncodeHex`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.EncodeHex.html
-[`EncodeJSON`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.EncodeJson.html
-[`Hash`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.Hash.html
-[`Random`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.Random.html
-[`ReadDir`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.ReadDir.html
-[`ReadEnv`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.ReadEnv.html
-[`ReadFile`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.ReadFile.html
-[`ReadStdin`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.ReadStdin.html
-[`SplitString`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.SplitString.html
-[`WriteFile`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.WriteFile.html
-[`WriteStderr`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.WriteStderr.html
-[`WriteStdout`]: https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.WriteStdout.html
+[`DecodeHex`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.DecodeHex.html>
+[`DecodeJSON`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.DecodeJson.html>
+[`Delay`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.Delay.html>
+[`Drop`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.Drop.html>
+[`Encode`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.Encode.html>
+[`EncodeCSV`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.EncodeCsv.html>
+[`EncodeHex`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.EncodeHex.html>
+[`EncodeJSON`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.EncodeJson.html>
+[`Hash`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.Hash.html>
+[`Random`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.Random.html>
+[`ReadDir`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.ReadDir.html>
+[`ReadEnv`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.ReadEnv.html>
+[`ReadFile`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.ReadFile.html>
+[`ReadStdin`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.ReadStdin.html>
+[`SplitString`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.SplitString.html>
+[`WriteFile`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.WriteFile.html>
+[`WriteStderr`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.WriteStderr.html>
+[`WriteStdout`]: <https://docs.rs/protoflow-blocks/latest/protoflow_blocks/struct.WriteStdout.html>
```

Comment:
What is happening here? There seems to be spurious `<` and `>` characters added to URLs here.

Reply:
fixed